js/bitstamp.js

```diff
@@ -572,6 +573,11 @@ module.exports = class bitstamp extends Exchange {
         return this.parseOrderStatus (this.safeString (response, 'status'));
     }
 
+    async fetchOrderTrades (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
```

Comment:
Ah, I thought you mean the base class ) Shouldn't we move this to the base class and in the process also add a check like:

```JavaScript
async fetchOrderTrades (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
    if (this.has['fetchOrder'] === true) {
        const order = await this.fetchOrder (id, symbol, params)
        return order['trades']
    }
}
```

What do you think?

Reply:
BTW, in this case the `fetchOrderTrades` becomes `'emulated'` via the `true`-endpoint for `fetchOrder`.